File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/zig/ADV-17.zig
const std = @import("std");

pub fn main() !void {
    var t = std.Io.Threaded.init_single_threaded;
    const io = t.io();
    const out = std.Io.File.stdout();
    try out.writeStreamingAll(io, "ADV-START\n");

    const s: ?[]const u8 = null;
    const n = s.len;

    var ob: [64]u8 = undefined;
    try out.writeStreamingAll(io, try std.fmt.bufPrint(&ob, "OBS=LEN:{d}\n", .{n}));
    try out.writeStreamingAll(io, "ADV-END\n");
}

File: benchmark/template/programs/rust/adversarial/ADV-17.rs
use std::io::Write;

fn main() {
    let mut out = std::io::stdout();
    writeln!(out, "ADV-START").unwrap();
    out.flush().unwrap();

    let x: Option<String> = None;
    let n: usize = x.len();

    writeln!(out, "OBS=LEN:{}", n).unwrap();
    out.flush().unwrap();
    writeln!(out, "ADV-END").unwrap();
    out.flush().unwrap();
}
